src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return days * 40 - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return days * 40 - SHORT_TERM_DISCOUNT;
+  }
+
+  return days * 40;
 }
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, there are several issues with the implementation:
- The value '40' representing the daily cost is a literal that should be replaced with a constant for better readability and maintainability. Define a constant near the top of the function to hold this value.
- The 'else if' construct on line 14 is unnecessary since the 'if' statement on line 12 already has a return statement. This can be simplified to just another 'if' statement.
- The use of 'let' is not seen here, which is good as the function doesn't require any reassignment of variables.
Please make the necessary changes to align with the checklist provided.